README.md

```diff
@@ -9,3 +9,39 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+## Answer
+
+There are two graphs $A$ and $B$ that are completely connected and both have $n$ vertices.
+
+We can have $[1,2,3, ..., n]$ represent the vertices of $A$ and $[a,b,c, ..., z]$ represent the vertices of $B$. 
+
+Since both graphs are completely connected, every vertex in $A$ is connected to every other vertex in $A$, and every vertex in $B$ is connected to every other vertex in $B$.
+
+We have a bijective mapping $M: V_A \rightarrow V_B$:
+
+
+1 $\rightarrow$ a
+
+2 $\rightarrow$ b
+
+3 $\rightarrow$ c
+
+⋮
+
+n $\rightarrow$ z
+
+This mapping $M$ is
+- One-to-one: Each vertex in $A$ maps to exactly one unique vertex in $B$
+- Onto: Every vertex in $B$ is mapped to by exactly one vertex in $A$
+  
+Therefore, $M$ satisfies the requirements of the isomorphism function $f$, it is a bijection between the vertex sets
```

Comment:
What about the other part of the definition? Are edges mapped to equivalent edges?

Reply:
I will address this